src/modules/link-selector.js

```diff
@@ -40,6 +40,10 @@
 			fakeAnchor.prop('href', t.attr('href'));
 			pathname = fakeAnchor.prop('pathname');
 
+			if (!!App.device.internetexplorer) {
```

Comment:
I think it would be better to check if the string starts with '/' or not, and add it if not.
```suggestion
			// IE respects the spec, for once...
			if (pathname.charAt(0) !== '/') {
```